code/modules/movespeed/modifiers/mobs.dm

```diff
@@ -161,3 +161,7 @@
 
 /datum/movespeed_modifier/goliath_mount
 	multiplicative_slowdown = -26
+
+/datum/movespeed_modifier/settler
```

Comment:
It doesn't look like it's being used at all.

Reply:
Oh, then I may have accidentally hit the space button while doing a quick ctrl+f search.